### PR TITLE
Add CVE-2020-13492

### PR DESCRIPTION
**Pull request by iamamoose**

### CODE REVIEW

File: 2020/13xxx/CVE-2020-13492.json [hunk 1]
```diff
@@ -1,18 +1,70 @@
 {
```
Comment (cve-team) [inline 2020/13xxx/CVE-2020-13492.json:1]: Invalid JSON file (invalid state)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by iamamoose** [comment]
Good catch, it validated (because the schema ignores this field).  I'll have to update our process to ensure it's put into PUBLIC before we export the json.

File: 2020/13xxx/CVE-2020-13492.json (hunk 1 above)
Comment (cve-team) [inline 2020/13xxx/CVE-2020-13492.json:1]: File name CVE ID and JSON CVE ID do not match.

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Pull request closed without merging**